message-sequence/message-sequence.bal

```diff
@@ -0,0 +1,38 @@
+import ballerina/io;
+import ballerina/http;
+
+type S3Response record {|
+    string x\-amz\-id;
+    string x\-amz\-request\-id;
+    string x\-amz\-server\-side\-encryption?;
+|};
+
+public function main() returns error? {
+    http:Client s3Client = check new ("http://asics-shoes.s3.amazonaws.com.balmock.io");
+
+    // Reads file as stream containing 4kb chunk.
+    stream<io:Block, io:Error?> fileStream = check io:fileReadBlocksAsStream("./resources/employee_names.txt", 4096);
+
+    map<string> req = {
+        x\-amz\-acl: "private",
+        x\-amz\-storage\-class: "STANDARD",
+        x\-amz\-checksum\-algorithm: "CRC32",
+        x\-amz\-server\-side\-encryption: "AES256"
+    };
+
+    http:Request request = new;
+
+    foreach var [key, value] in req.entries() {
```

Comment:
Why can't you directly add the headers without iteration over a map of headers?

Reply:
If you are going to use this map it's better to rename to something like `s3Headers`.

Reply:
Calling `addHeader` multiple lines seems not appropriate. I will update the name